src/offer_call_builder.ts

```diff
@@ -41,7 +41,8 @@ export class OfferCallBuilder extends CallBuilder<
    * @returns {OfferCallBuilder} current OfferCallBuilder instance
    */
   public forAccount(id: string): this {
-    return this.forEndpoint("accounts", id);
+    this.url.setQuery("seller", id);
```

Comment:
Huh, good point.. took a while for this to come up! Thanks for the quick fix :+1:

Reply:
Oh hm, after looking at the tests it looks like this is actually a different endpoint than you expected: this resolves to `GET /accounts/:id/offers`. I think what we want to do is add a new `public seller(id: string): this`, instead.